tests/Microsoft.Bot.Schema.Tests/Teams/MessageActionsPayloadTests.cs

```diff
@@ -0,0 +1,388 @@
+        /// </summary>
+        [Fact]
+        public void TestGetId()
+        {
+            var id = "testId";
+            var activity = new Activity
+            {
+                Type = ActivityTypes.Invoke,
+                Name = "composeExtension/submitAction",
+                ChannelData = new MessagingExtensionAction { MessagePayload = new MessageActionsPayload { Id = id } }
+            };
+
+            var channelData = activity.GetChannelData<MessagingExtensionAction>();
+
+            Assert.Equal(id, channelData.MessagePayload.Id);
+        }
+
+        /// <summary>
+        /// Ensures that the ReplyToId property can be set and retrieved.
+        /// </summary>
+        [Fact]
+        public void TestGetReplyToId()
+        {
+            var replyToId = "replyId";
+            var activity = new Activity
+            {
+                Type = ActivityTypes.Invoke,
+                Name = "composeExtension/submitAction",
+                ChannelData = new MessagingExtensionAction { MessagePayload = new MessageActionsPayload { ReplyToId = replyToId } }
+            };
+
+            var channelData = activity.GetChannelData<MessagingExtensionAction>();
+
+            Assert.Equal(replyToId, channelData.MessagePayload.ReplyToId);
+        }
+
+        /// <summary>
+        /// Ensures that the MessageType property can be set and retrieved.
+        /// </summary>
+        [Fact]
+        public void TestGetMessageType()
+        {
+            var messageType = "message";
+            var activity = new Activity
+            {
+                Type = ActivityTypes.Invoke,
+                Name = "composeExtension/submitAction",
+                ChannelData = new MessagingExtensionAction { MessagePayload = new MessageActionsPayload { MessageType = messageType } }
+            };
+
+            var channelData = activity.GetChannelData<MessagingExtensionAction>();
+
+            Assert.Equal(messageType, channelData.MessagePayload.MessageType);
+        }
+
+        /// <summary>
+        /// Ensures that the CreatedDateTime property can be set and retrieved.
+        /// </summary>
+        [Fact]
+        public void TestGetCreatedDateTime()
+        {
+            var createdDateTime = DateTime.Today.ToString();
+            var activity = new Activity
+            {
+                Type = ActivityTypes.Invoke,
+                Name = "composeExtension/submitAction",
+                ChannelData = new MessagingExtensionAction { MessagePayload = new MessageActionsPayload { CreatedDateTime = createdDateTime } }
+            };
+
+            var channelData = activity.GetChannelData<MessagingExtensionAction>();
+
+            Assert.Equal(createdDateTime, channelData.MessagePayload.CreatedDateTime);
+        }
+
+        /// <summary>
+        /// Ensures that the LastModifiedDateTime property can be set and retrieved.
+        /// </summary>
+        [Fact]
+        public void TestGetLastModifiedDateTime()
```

Comment:
This test seems to be testing MessagingExtensionAction, the focus of this test class is to test MessageActionPayload, I would suggest we remove it.

Reply:
Done. Thanks!